magento/magento2
# Fix minisearch not appear when disable suggestions search

### Description (*)

Previously when owner shop disable suggestions search in backend config when user try search in mobile screen. Mini search will not active. User will not able to search thing

Use declarative syntax `data-mage-init` for improve performance instead script tag `x-magento-init`

### Preconditions (*)

Magento 2.4-develop
Php 7.4

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#30584
### Manual testing scenarios (*)

1. Go to admin
2. Catalog > Catalog Search 
3. Enable Search Suggestions Change to No
4. View storefront in real device mobile screen or resize browser screen
5. Click on search icon to see problem

### Expected result (*)

 Appear input form search (form minisearch)


### Actual result (*)
:heavy_multiplication_x: Nothing happens

![Peek 2020-10-22 12-37](https://user-images.githubusercontent.com/51679138/96853715-5aa4ce00-1463-11eb-848c-361fd2ec8ea2.gif)
### Questions or comments


### Contribution checklist (*)
